index1.html

```diff
@@ -1,14 +1,63 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <link type="text/css" rel="stylesheet" href="./styles.css"/>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Movie Bonanza </title>
 </head>
 <body>
+    <div id="header"><h1>Working Title</h1></div>
+    <div class="row" id="container">
+    <div class="col s3"></div>
+    <div class="col s2" id="facePlate1">
+        <img src="./imageFrame.png">       
+    </div>
+    <div class="col s2" id="posterPlate">
+        <img src="./imageFrame.png">       
+    </div>
+    <div class="col s2" id="facePlate2">
+        <img src="./imageFrame.png">       
+    </div>
+           
+    </div>
+  
 
-    This a;sdjfa'iojf'ewi;oa
     
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+      <!--JavaScript at end of body for optimized loading-->
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
+      
+
+
+
+
+
+
+
+<!-- <div id="container">
+    <div class="valign-wrapper" id="facePlate1">
+        <img src="./imageFrame.png">          
+    </div>
+    <div id="posterPlate1"><img src="./imageFrame.png"></div>
+    <div id="facePlate2"><img src="./imageFrame.png"></div>
+</div> -->
+
```

Comment:
this is outside the html ? for good reason ? 
no conflicts so i am going to approve the pull but i believe we need to move and or delete this.

Reply:
You are correct.  I pulled that little snippet out whilst experimenting with materialize's alignment helpers.  It's been deleted, though.